From f9099189bf875511a76ea338b32b918330d5950f Mon Sep 17 00:00:00 2001
From: rina <rina7827singh@gmail.com>
Date: Tue, 13 Mar 2018 00:22:49 +0530
Subject: [PATCH] change

---
 new 1.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/new 1.txt b/new 1.txt
index c6b3c5d..85df507 100644
--- a/new 1.txt	
+++ b/new 1.txt	
@@ -1 +1 @@
-adsfdfgjhjlj;kl;ko
\ No newline at end of file
+abcd
\ No newline at end of file
-- 
2.16.2.windows.1

